### ISSUE TITLE
Debugging 

### ISSUE THREAD

**Issue by Andr3wjTV** [takeshixx/redstar-tools#11]
When i started to debug Red Star Os and i wrote this on terminal /media/RedStar\ Desktop\ 3.0/RedStar/RPMS , he tell me " no such file or directory" How can i resolve this

**Comment by takeshixx** [comment]
Have you mounted the disc?

**Comment by Andr3wjTV** [comment]
Yes, i mounted the disk on virtualbox, and i execute defuse.sh on Red Star Os and the red flag on the frame at the top right Is disappeared.
